CHANGELOG.md

```diff
@@ -3,6 +3,10 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
+## [2.60.0]
```

Comment:
bear in mind that this need to refer to the version yet to come, so it should be 2.61.0